### PR TITLE
feat(issues) Add skeleton for Org wide issues

### PR DESCRIPTION
**Pull request by markstory**
Its a place to start. I wanted to get feedback on the name `organizationStream`.

At a high level I was thinking of duplicating most of the stream components as there are subtle shifts in requirements like:

* Bulk operations will only work if issues from a single project are selected.
* Search context comes from both the global filter bar and other search controls.
* Saved Searches come from different endpoints and behave differently in the search query input.

I've added a few placeholder components that I plan on replacing individually as the APIs for them become available. Notably absent from these changes are saved searches and processing issues which will be added later as they have their own tasks in Jira.

Refs APP-990

### CODE REVIEW

File: src/sentry/static/sentry/app/routes.jsx [hunk 1]
```diff
@@ -777,12 +775,10 @@ function routes() {
             <Redirect path="saved/" to="/organizations/:orgId/discover/" />
             <Route path="saved/:savedQueryId/" />
           </Route>
-
           <Route
             path="/organizations/:orgId/activity/"
             component={errorHandler(OrganizationActivity)}
           />
-
```
Comment (markstory) [inline src/sentry/static/sentry/app/routes.jsx]: Prettier doing its thing 🤷‍♂️

File: src/sentry/static/sentry/app/routes.jsx [hunk 2]
```diff
@@ -795,14 +791,38 @@ function routes() {
...
+              import(/* webpackChunkName: "OrganizationStreamContainer" */ './views/organizationStream/container')}
+            component={errorHandler(LazyLoad)}
+          >
+            <IndexRoute
+              componentPromise={() =>
+                import(/* webpackChunkName: "OrganizationStreamOverview" */ './views/organizationStream/overview')}
+              component={errorHandler(LazyLoad)}
+            />
+          </Route>
+          /* Once org issues is complete, these routes can be nested under
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: Is this a valid comment? Doesn't it need `{}`?

File: src/sentry/static/sentry/app/routes.jsx [hunk 3]
```diff
@@ -795,14 +791,38 @@ function routes() {
...
-
+          <Route
+            path="/organizations/:orgId/issues/"
+            componentPromise={() =>
+              import(/* webpackChunkName: "OrganizationStreamContainer" */ './views/organizationStream/container')}
+            component={errorHandler(LazyLoad)}
+          >
+            <IndexRoute
+              componentPromise={() =>
+                import(/* webpackChunkName: "OrganizationStreamOverview" */ './views/organizationStream/overview')}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: I think the name is good - Is the stream something we'll want to keep codesplit once it's live on prod? We probably want it in the main bundle (and codesplit the older stream).

Comment (markstory) [reply src/sentry/static/sentry/app/routes.jsx]: Yeah once org issues are out they should probably join the main bundle but for now I didn't want to bog down customers with code they won't use. I added APP-992 to track the removal of the code split.

**Review by lynnagara — COMMENTED** [review]
Assume this is mostly copied from the stream component, might be nice though not essential to share some of the code since this may be around for a while. I think the environments stuff might need to be changed since the way we handle that now in the org views (global selection store) is different from how environment filtering worked in the project specific views

**Comment by markstory** [comment]
@lynnagara The differences in how environment parameters work is making me think that a copy is going to be better for this page and the search/bulk actions.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**